_sass/minima/dark-mode.scss

```
@@ -82,3 +82,13 @@ table td{
 canvas {
     filter: invert(100%);
 }
+
+.post-list > li > div {
+    box-shadow: none !important;
```

Inherently  *fastpages* uses card syle when `show_image` is enabled, can be seen [here](https://github.com/fastai/fastpages/blob/master/_fastpages_docs/_show_image_true.png). I made some of those changes in order to be consistent with the default design aesthetics.
I already *love your design aesthetics* and the suggested changes look amazing.

---

Thanks for raising this point.
If you check out Material design [guidelines](https://material.io/design/color/dark-theme.html#usage) for Dark theme, card styles have a _lighter_ background colour to portray elevation, as opposed to box shadows in light themes.
I've tried this out and it looks nice. Let me know what you think.

```suggestion
    background-color: $overlay;
```

---

Also everything else looks in order. Thank you :D

---

I agree, cards without shadow do look great in Dark Mode. I use the same design for my blog.